src/app/components/profile/profile.component.ts

```diff
@@ -171,6 +171,30 @@ export class ProfileComponent implements OnInit {
           placeholder: 'Affiliated To',
           type: 'text',
           value: this.user['affiliation']
+        },
+        {
+          isRequired: true,
```

Comment:
Can we make the field optional? Maybe the user doesn't have a Google Scholar URL.